scripts/import_suppliers.py

```diff
@@ -0,0 +1,298 @@
+        if not string:
+            return []
+
+        return [value for value in
+                map(unicode.strip, string.split(',')) if len(value) is not 0]
+
+    @staticmethod
+    def un_nest_key_value_pairs(obj, key_of_nested_obj):
+        if key_of_nested_obj not in obj.keys():
+            return obj
+
+        nested_obj = obj[key_of_nested_obj]
+
+        obj = drop_foreign_fields(
+            obj,
+            [key_of_nested_obj]
+        )
+
+        for key in nested_obj.keys():
+            obj[key] = nested_obj[key]
+
+        return obj
+
+    @staticmethod
+    def nest_key_value_pairs(base_obj, key_for_new_obj, keys_to_nest):
+
+        base_obj[key_for_new_obj] = {}
+
+        for key_to_nest in keys_to_nest:
+            if key_to_nest in base_obj.keys():
+                # Maybe an else if the key isn't here.
+                base_obj[key_for_new_obj][key_to_nest] = \
+                    base_obj.pop(key_to_nest)
+
+        return base_obj
+
+    @staticmethod
+    def change_key_names(obj, list_of_lists_of_key_pairs):
+
+        for key_pair in list_of_lists_of_key_pairs:
+            if len(key_pair) is 2:
+                obj = SupplierPutter.change_key_name(
+                    obj,
+                    key_pair[0],
+                    key_pair[1]
+                )
+
+        return obj
+
+    @staticmethod
+    def change_key_name(obj, old_key, new_key):
+        if old_key in obj.keys():
+            obj[new_key] = obj.pop(old_key)
+
+        return obj
+
+    @staticmethod
+    def convert_a_json_object_into_an_array_with_one_entry(obj):
+
+        temp = [{}]
+
+        for key in obj.keys():
+            temp[0][key] = obj.pop(key)
+
+        obj = temp
+
+        return obj
+
+
+# this is copied + pasted from app/main/utils.py
+def drop_foreign_fields(json_object, list_of_keys):
+    json_object = json_object.copy()
+    for key in list_of_keys:
+        json_object.pop(key, None)
+
+    return json_object
+
+
+def do_import(base_url, access_token, filename, cert, verbose):
+    endpoint = "{}/suppliers".format(base_url)
+    print("Base URL: {}".format(base_url))
+    print("Access token: {}".format(access_token))
+    print("Filename: {}".format(filename))
+
+    putter = SupplierPutter(endpoint, access_token, cert)
+
+    counter = 0
+    start_time = datetime.now()
+
+    with open(filename) as data_file:
+        try:
+            json_from_file = json.load(data_file)
+        except ValueError:
+            print("Skipping {}: not a valid JSON file".format(filename))
+
+    for supplier in json_from_file['suppliers']:
+        supplier_id, response = putter.import_supplier(supplier)
+
+        if response is None:
+            print("ERROR: Supplier (id={}) not imported".format(supplier_id),
+                  file=sys.stderr)
+        elif response.status_code / 100 != 2:
+            print("ERROR: {} on Supplier (id={})".format(response.status_code, supplier_id),
```

Comment:
Line too long! If you can make Travis go green we should be able to get this merged today.